### PR TITLE
FE-972: Hibana style cleanup for onboarding pages

### PR DESCRIPTION
**Pull request by bkemper**
### What Changed
 - Hibana'd onboarding pages
 - Refactored `PlanPicker` to be restyleable
*** Please note that RecipientValidationPriceTable updates will be provided by #1601 

### How To Test

**The sending workflow:**
 - Visit /join, 
 - Fill-out and submit form [takes you to /onboarding/plan]
 - Confirm form looks good and functions correctly for both free and paid plans
 - Pick a plan [takes you to /onboarding/sending-domain]
 - Confirm form looks good and functions correctly

**The recipient validation workflow:**
 - Visit /join?product=rv
 - Fill-out and submit form [takes you to /onboarding/recipient-validation]
 - Confirm form looks good and functions correctly

### CODE REVIEW

File: src/pages/onboarding/RVBundlePage.js [hunk 1]
```diff
@@ -8,98 +8,94 @@ import {
...
-      <h1 className={styles.header}>You're Almost There...</h1>
-      <p className={styles.subheader}>
-        Here at SparkPost we do it all! As part of our Recipient Validation service we also provide
-        a SparkPost sending account. Make every email count and start verifying.
-      </p>
+    <>
+      <div className={styles.Header}>
+        <Stack>
+          <CenteredLogo />
+          <Heading as="h2">You're Almost There...</Heading>
```
[the remark this answers is not in the document]
Comment (bkemper) [reply src/pages/onboarding/RVBundlePage.js]: Done... 91b5a1a

File: src/components/planPicker/PlanPickerHibana.module.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+.PlanPickerHelpText {
+  font-size: font-size(200);
+  font-weight: font-weight(light);
+  padding: 0 spacing(400) spacing(400);
+}
+
+.PlanPickerInput {
+  // todo?
+  // @include visually-hidden;
```
[the remark this answers is not in the document]
Comment (bkemper) [reply src/components/planPicker/PlanPickerHibana.module.scss:20]: As we discussed, it is an input, so I set `type="hidden"`.

File: src/components/planPicker/PlanPickerHibana.module.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  border-radius: 0;
+  box-shadow: box-shadow(400);
+  max-height: sizing(1000);
+  // todo?
+  // transform: translate(0, -18px);
+
+  &.open {
+    // todo?
+    // transform: translate(0, 0);
+  }
```
[the remark this answers is not in the document]
Comment (bkemper) [reply src/components/planPicker/PlanPickerHibana.module.scss:29]: Agreed, a little too much.

**Comment by bkemper** [comment]
@nicklemmon or @nstanard This should be ready for another pass.

File: src/components/planPicker/PlanPicker.js [hunk 4]
```diff
@@ -19,8 +22,11 @@ const TIERS = [
 /**
  * This component will register the a redux-form field named 'planpicker'
  * Entire selected plan object is stored in state
+ *
+ * todo, this should be grouped with other redux-form wrappers
  */
-export function PlanPicker({ bundles, input, disabled, selectedPromo }) {
+export function PlanPicker({ bundles, input, selectedPromo }) {
```
Comment (nicklemmon) [inline src/components/planPicker/PlanPicker.js:28]: The HTML rendered in the dropdown doesn't really make much sense - at the same time, now isn't the time to address that. If we ever get a good chance to improve a11y, this is a good component to address.

Comment (nicklemmon) [reply src/components/planPicker/PlanPicker.js:28]: Downshift can do a lot of the work for us, but I don't think we are leveraging all of it:
https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

I know we discussed trying `react-select` at some point too.

Some other time perhaps!

File: src/components/planPicker/PlanPicker.js [hunk 5]
```diff
@@ -56,67 +62,57 @@ export function PlanPicker({ bundles, input, disabled, selectedPromo }) {
...
-            Interested in learning more about our Starter and Premier plans? Check out our{' '}
-          </span>
+            Interested in learning more about our Starter and Premier plans? Check out our
+          </span>{' '}
           <ExternalLink to="https://www.sparkpost.com/docs/faq/difference-between-starter-and-premier">
             Knowledge Base
           </ExternalLink>
-        </div>
+        </p>
+        <input {...getInputProps()} ref={inputRef} readOnly type="hidden" />
```
Comment (nicklemmon) [inline src/components/planPicker/PlanPicker.js:115]: Nice. 👍

File: src/components/planPicker/PlanPickerHibana.module.scss [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  max-height: sizing(1000);
+  transform: translateY(-3px);
+
+  &.open {
+    transform: translateY(0);
+  }
+}
+
+.PlanPickerOption {
+  @include button-reset();
```
Comment (nicklemmon) [inline src/components/planPicker/PlanPickerHibana.module.scss:31]: We could potentially add an `<UnstyledButton/>` component some day

File: src/pages/onboarding/RVBundlePage.js [hunk 7]
```diff
@@ -8,98 +8,96 @@ import {
...
-        Here at SparkPost we do it all! As part of our Recipient Validation service we also provide
-        a SparkPost sending account. Make every email count and start verifying.
-      </p>
+    <>
+      <div className={styles.Header}>
+        <Stack>
+          <CenteredLogo />
+          <Heading as="h1" looksLike="h2">
+            You're Almost There...
+          </Heading>
```
Comment (nicklemmon) [inline src/pages/onboarding/RVBundlePage.js:40]: 👍

File: src/pages/onboarding/RVBundlePage.js [hunk 8]
```diff
@@ -8,98 +8,96 @@ import {
...
+                  Recipient Validation
+                </CardTitle>
+                <CardContent>
+                  <Stack>
+                    <p>
+                      With our monthly pay-as-you-go plan using tiered pricing, the more you
+                      validate, the less you pay per message.
+                    </p>
+                    <p>
+                      <UnstyledLink onClick={() => setModalOpen(true)}>See Pricing</UnstyledLink>
```
Comment (nicklemmon) [inline src/pages/onboarding/RVBundlePage.js:67]: This won't be interactive with a keyboard. This should do it:

```js
const handleSeePricingClick = e => {
  e.preventDefault();
  setModalOpen(true);
};

<UnstyledLink
  title="Opens a dialog"
  role="button"
  href="#"  // use `to` if `href` doesn't work
  onClick={handleSeePricingClick}
>
  See Pricing
</UnstyledLink/>
```

By using `e.preventDefault()` it should prevent any page scrolling that comes with the `href`, but still allow for keyboard focus.

**Pull request merged**